lib/paypal-sdk/rest/data_types.rb

```
@@ -1300,7 +1300,12 @@ def get_cert_chain()
           end
 
           def get_expected_sig(transmission_id, timestamp, webhook_id, event_body)
-            crc = Zlib::crc32(event_body).to_s
+            begin 
+              utf8_encoded_event_body = event_body.encode("UTF-8")
+              crc = Zlib::crc32(utf8_encoded_event_body).to_s
+            rescue Encoding::UndefinedConversionError
+              crc = ""

```

Updated to just let the error propagate in the hopefully rare case where a string can't be encoded into UTF-8.